src/lib/containers/query_wrapper_plot_nav/QueryWrapperPlotNav.tsx

```diff
@@ -131,14 +133,19 @@ const QueryWrapperPlotNav: React.FunctionComponent<QueryWrapperPlotNavProps> =
                       sql={sqlUsed}
                       hideDownload={hideDownload}
                       hideQueryCount={hideQueryCount}
+                      hideFacetFilterControl={!isFaceted}
+                      hideVisualizationsControl={!isFaceted}
                     />
-                    <QueryFilter {...queryWrapperChildProps} {...props} />
-                    <QueryFilterToggleButton {...queryWrapperChildProps} />
+                    {isFaceted && <>
+                      <QueryFilter {...queryWrapperChildProps} {...props} />
+                      <QueryFilterToggleButton {...queryWrapperChildProps} />
+                      </>}
                     <FacetNav
```

Comment:
This is a little strange that we still show the FacetNav, but it also contains the query count, and shows the conditions that have been applied.